api/server/sdk/volume.go

```diff
@@ -0,0 +1,30 @@
+/*
+Package sdk is the gRPC implementation of the SDK gRPC server
+Copyright 2018 Portworx
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package sdk
+
+import (
+	"github.com/libopenstorage/openstorage/api/spec"
+	"github.com/libopenstorage/openstorage/cluster"
+	"github.com/libopenstorage/openstorage/volume"
+)
+
+// VolumeServer is an implementation of the gRPC OpenStorageVolume interface
+type VolumeServer struct {
+	specHandler spec.SpecHandler
+	driver      volume.VolumeDriver
```

Comment:
Current OSD http server handles multiple implementations of drivers simultaneously. It looks at the http request and chooses the correct driver. Are we planning to support that in grpc ?

Reply:
I have not seen a use case for this in openstorage, but if it turns out to be one, we will need can discuss it. Right now, just like CSI, the gRPC SDK will support only one driver.